src/js/form-builder.js

```diff
@@ -1702,6 +1706,15 @@ function FormBuilder(opts, element, $) {
     $valWrap.toggle(e.target.value !== 'quill')
   })
 
+  const testForm = document.createElement('form')
+  $stage.on('change', '[name="name"]', e => {
+    const name = e.target.value
+    if (name in document || name in testForm) {
+      //@TODO Notify the user of this potential issue
```

Comment:
i dont quite understand the issue here. wouldn't we _want_ to dom clobber names out of the html?

Reply:
From https://whatpr.org/whatwg_html/8474/form-control-infrastructure.html#attributes-common-to-form-controls

> DOM clobbering is a common cause of security issues. Avoid using the names of built-in form properties with the [name](https://whatpr.org/whatwg_html/8474/form-control-infrastructure.html#attr-fe-name) and id content attributes.

Say if you name your Text input 'method' when we render this field in formRender (not preview since fields name have a suffix -preview appended) then the containing form's method attribute will be clobbered with the text input

Reply:
i see. thanks for explaining